packages/dom/README.md

```diff
@@ -272,7 +272,7 @@ _Parameters_
 
 -   _container_ `HTMLElement`: Focusable element.
 -   _isReverse_ `boolean`: True for end, false for start.
--   _mayUseScroll_ `boolean`: Whether to allow scrolling.
+-   _mayUseScroll_ `[boolean]`: Whether to allow scrolling.
```

Comment:
Should we also document the default value here?

Reply:
The generated documentation doesn't pull out the default value 😞 I'm not sure it's even able to, if you go to the [comment-parser playground](https://syavorsky.github.io/comment-parser/#parse-defaults) and replace the comments there with the following you can check out the AST it generates, it only pulls the name out and whether it is optional but not the supplied default value (unless you dig into the `source` tree)


```js
/**
 * @param {boolean} [foo=false] Description
 */

const parsed = parse(source);
const stringified = parsed.map((block) => stringify(block));
```

Reply:
Guess we can add it manually? But yeah, that's a mess. Feel free to ignore it anyway 👍

Reply:
If we add it manually it'll get erased by `docgen` the next time it runs (and the CI will complain that the docs are out of date).